server/Dockerfile

```
@@ -206,6 +206,10 @@ RUN ARCH=$(uname -m) && \
 # Install Tailscale CLI (for local dev SSH to tailnet devices)
 RUN curl -fsSL https://tailscale.com/install.sh | sh
 
+# Install Fly.io CLI (flyctl) for Fly.io connector
+ARG FLYCTL_VERSION=0.4.57
+RUN curl -fsSL "https://fly.io/install.sh" | FLYCTL_INSTALL="/usr/local" FLYCTL_VERSION=${FLYCTL_VERSION} sh
```

Hadolint (2.14.0)

[warning] Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check

(DL4006)